src/infrastructure/audit/api.js

```diff
@@ -1,10 +1,9 @@
-const rp = require('login.dfe.request-promise-retry');
+const { fetchApi } = require('login.dfe.async-retry');
 
 const updateAuditLogs = async () => {
   try {
-    await rp({
+    await fetchApi(`${process.env.AUDIT_HTTP_TRIGGER_URL}`, {
       method: 'POST',
-      uri: `${process.env.AUDIT_HTTP_TRIGGER_URL}`,
       json: true,
```

Comment:
Similar to above; `json: true` isn't needed anymore.

Reply:
Updated